lib/archethic/mining/distributed_workflow.ex

```diff
@@ -189,7 +189,10 @@ defmodule Archethic.Mining.DistributedWorkflow do
       else
         resolved_addresses
         |> Enum.map(fn {_origin, resolved} -> resolved end)
-        |> Enum.concat([LedgerOperations.burning_address()])
+        |> maybe_add_burning_address(
```

Comment:
I don't get the need of adding potentially the burning address.
If it's already in the movements, the address will be mentioned in the resolved addresses.
But we can avoid fetching the last address of the burn address in the `TransactionChain.resolve_transaction_addresses`

Reply:
Addressed avoir fetching the last address of the burn addresses in commit : [82a4a00](https://github.com/archethic-foundation/archethic-node/pull/749/commits/82a4a007e5afea0d4c9f751a0bfd1f336e30287d)

Reply:
I still think this `maybe_add_burning_address` is not required

Reply:
sorry, it is removed now.